itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestTableLevelReplicationScenarios.java

```diff
@@ -313,8 +313,8 @@ public void testIncorrectTablePolicyInReplDump() throws Throwable {
     }
     Assert.assertTrue(failed);
 
-    // Invalid pattern where we didn't enclose table pattern within single or double quotes.
-    replPolicy = primaryDbName + ".[t1].[t2]";
+    // Invalid pattern where where table list is empty.
+    replPolicy = primaryDbName + ".''.'t2'";
```

Comment:
Check empty list for exclude list as well. Also add a test with more than 2 dots in the policy.

Reply:
done